pymodbus/client/base.py

```diff
@@ -121,7 +121,7 @@ def __init__(
         self.params.kwargs = kwargs
 
         # Common variables.
-        self.framer = self.params.framer(ClientDecoder(), self)
+        self.framer = self.params.framer(ClientDecoder(), self)  # type: ignore
```

Comment:
Please have the whole file ignored, since there will be a lot of changes over the next short time.

Reply:
I think it makes sense for whomever is working on a file to
(1) don't make type errors OR
(2) disable the type checking temporarily during the refactor.

This can be done by adding `# type: ignore` of the file, I believe.

Reply:
Then please do that.

Reply:
I've left this below in the config so you can see how to ignore files.  It's not needed currently.
```
[mypy]
# exclude = pymodbus/client/base.py
```

Does this work for you?